backend/lib/services/tickets.js

```diff
@@ -21,10 +21,12 @@ function fromLabel (item) {
   ])
 }
 
-const converter = exports.converter = markdown.createConverter({
-  ghMentions: true,
-  ghMentionsLink: new URL(config.gitHub.apiUrl).origin + '/{u}'
-})
+const converter = exports.converter = markdown.createConverter(
+  _.get(config, 'gitHub.apiUrl') ? {
+    ghMentions: true,
+    ghMentionsLink: new URL(config.gitHub.apiUrl).origin + '/{u}'
+  } : { }
+)
```

Comment:
extract options to variable to make it more readable
```js
const options = {}
const apiUrl = _.get(config, 'gitHub.apiUrl')
if (apiUrl) {
  options.ghMentions = true
  options.ghMentionsLink = new URL(apiUrl).origin + '/{u}'
}
const converter = exports.converter = markdown.createConverter(options)
```

Reply:
Will do so.